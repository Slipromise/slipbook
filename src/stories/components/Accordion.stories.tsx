import React from "react";
import { ComponentStory, ComponentMeta } from "@storybook/react";

import Accordion from "react-bootstrap/Accordion";

export default {
  component: Accordion,
} as ComponentMeta<typeof Accordion>;

const Template: ComponentStory<typeof Accordion> = (props) => (
  <Accordion {...props}>
    <Accordion.Item eventKey="0">
      <Accordion.Header>Accordion Item #1</Accordion.Header>
      <Accordion.Body>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
        veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
        commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
        velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint
        occaecat cupidatat non proident, sunt in culpa qui officia deserunt
        mollit anim id est laborum.
      </Accordion.Body>
    </Accordion.Item>
    <Accordion.Item eventKey="1">
      <Accordion.Header>Accordion Item #2</Accordion.Header>
      <Accordion.Body>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
        veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
        commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
        velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint
        occaecat cupidatat non proident, sunt in culpa qui officia deserunt
        mollit anim id est laborum.
      </Accordion.Body>
    </Accordion.Item>
  </Accordion>
);

export const Basic = Template.bind({});

Basic.args = {
  defaultActiveKey: "0",
};

export const FullyCollapsedState = Template.bind({});

export const Flush = Template.bind({});

Flush.args = {
  defaultActiveKey: "0",
  flush: true,
};

export const AlwaysOpen = Template.bind({});

AlwaysOpen.args = {
  defaultActiveKey: "0",
  alwaysOpen: true,
};

// TODO: useAccordionButton
