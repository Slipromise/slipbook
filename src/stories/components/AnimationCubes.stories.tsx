import AnimationCubes from "@/components/AnimationCubes";
import type { Meta, StoryObj } from "@storybook/react";

const meta: Meta<typeof AnimationCubes> = {
  component: AnimationCubes,
  parameters: {
    layout: "centered",
  },
};
export default meta;

type Story = StoryObj<typeof AnimationCubes>;

export const Normal: Story = {
  args: {
    updateRate: 500,
    duration: 2000,
  },
};

export const FastUpdates: Story = {
  args: {
    updateRate: 250,
    duration: 4000,
  },
};
