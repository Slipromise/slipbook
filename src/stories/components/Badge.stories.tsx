import React from "react";
import { ComponentStory, ComponentMeta } from "@storybook/react";

import Badge from "react-bootstrap/Badge";

export default {
  component: Badge,
} as ComponentMeta<typeof Badge>;

const Template: ComponentStory<typeof Badge> = (props) => (
  <div>
    <Badge {...props} bg="primary">
      Primary
    </Badge>{" "}
    <Badge {...props} bg="secondary">
      Secondary
    </Badge>{" "}
    <Badge {...props} bg="success">
      Success
    </Badge>{" "}
    <Badge {...props} bg="danger">
      Danger
    </Badge>{" "}
    <Badge {...props} bg="warning" text="dark">
      Warning
    </Badge>{" "}
    <Badge {...props} bg="info">
      Info
    </Badge>{" "}
    <Badge {...props} bg="light" text="dark">
      Light
    </Badge>{" "}
    <Badge {...props} bg="dark">
      Dark
    </Badge>
  </div>
);

export const Basic = Template.bind({});

export const Pill = Template.bind({});

Pill.args = { pill: true };
