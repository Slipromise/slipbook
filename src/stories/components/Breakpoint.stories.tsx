import React from "react";
import { ComponentMeta } from "@storybook/react";

import ThemeProvider from "react-bootstrap/ThemeProvider";

export default {
  component: ThemeProvider,
} as ComponentMeta<typeof ThemeProvider>;

export const Breakpoint = () => (
  <ThemeProvider
    breakpoints={["xxxl", "xxl", "xl", "lg", "md", "sm", "xs", "xxs"]}
    minBreakpoint="xxs"
  >
    <div>Your app...</div>
  </ThemeProvider>
);
