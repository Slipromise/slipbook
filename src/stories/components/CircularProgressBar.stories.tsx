import CircularProgressBar from "@/components/CircularProgressBar";
import type { Meta, StoryObj } from "@storybook/react";

const meta: Meta<typeof CircularProgressBar> = {
  component: CircularProgressBar,
  tags: ["autodocs"],
};

export default meta;

type Story = StoryObj<typeof CircularProgressBar>;

export const Normal: Story = {
  args: {
    value: 40,
  },
};

export const WithTitles: Story = {
  args: {
    value: 50,
    title: "React",
  },
};

export const Colorize: Story = {
  args: {
    value: 60,
    title: "React",
    color: "red",
  },
};
