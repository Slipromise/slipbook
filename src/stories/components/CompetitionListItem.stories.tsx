import type { Meta, StoryObj } from "@storybook/react";
import CompetitionListItem from "@/components/CompetitionListItem";

import React from "react";
import { withHolder } from "@/components/withHolder";
import { fn } from "@storybook/test";

const meta: Meta<typeof CompetitionListItem> = {
  component: CompetitionListItem,
  decorators: [
    (story) => {
      const Component = withHolder(story);
      return <Component></Component>;
    },
  ],
  argTypes: { startTime: { control: "date" } },
  tags: ["autodocs"],
  args: {
    onClick: fn(),
    onSubscript: fn(),
  },
};

export default meta;

const images = [
  "data:image/svg+xml;charset=UTF-8,%3Csvg%20width%3D%2264%22%20height%3D%2264%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%2064%2064%22%20preserveAspectRatio%3D%22none%22%3E%3Cdefs%3E%3Cstyle%20type%3D%22text%2Fcss%22%3E%23holder_18f57ed3b02%20text%20%7B%20fill%3A%231C2846%3Bfont-weight%3Abold%3Bfont-family%3AArial%2C%20Helvetica%2C%20Open%20Sans%2C%20sans-serif%2C%20monospace%3Bfont-size%3A10pt%20%7D%20%3C%2Fstyle%3E%3C%2Fdefs%3E%3Cg%20id%3D%22holder_18f57ed3b02%22%3E%3Crect%20width%3D%2264%22%20height%3D%2264%22%20fill%3D%22%23F8591A%22%3E%3C%2Frect%3E%3Cg%3E%3Ctext%20x%3D%2213.4609375%22%20y%3D%2236.5%22%3E64x64%3C%2Ftext%3E%3C%2Fg%3E%3C%2Fg%3E%3C%2Fsvg%3E",
  "data:image/svg+xml;charset=UTF-8,%3Csvg%20width%3D%2280%22%20height%3D%2280%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%2080%2080%22%20preserveAspectRatio%3D%22none%22%3E%3Cdefs%3E%3Cstyle%20type%3D%22text%2Fcss%22%3E%23holder_18f57f77702%20text%20%7B%20fill%3A%23FFFFFF%3Bfont-weight%3Abold%3Bfont-family%3AArial%2C%20Helvetica%2C%20Open%20Sans%2C%20sans-serif%2C%20monospace%3Bfont-size%3A10pt%20%7D%20%3C%2Fstyle%3E%3C%2Fdefs%3E%3Cg%20id%3D%22holder_18f57f77702%22%3E%3Crect%20width%3D%2280%22%20height%3D%2280%22%20fill%3D%22%233a5a97%22%3E%3C%2Frect%3E%3Cg%3E%3Ctext%20x%3D%2221.4609375%22%20y%3D%2244.5%22%3E80x80%3C%2Ftext%3E%3C%2Fg%3E%3C%2Fg%3E%3C%2Fsvg%3E",
];

const times = [1715167999872];

type Story = StoryObj<typeof CompetitionListItem>;

export const Normal: Story = {
  args: {
    title: "Title",
    status: "NONE",
    startTime: times[0],
    aTeamName: "ATeam",
    bTeamName: "BTeam",
    aTeamAvatarSrc: images[0],
    bTeamAvatarSrc: images[0],
  },
};

export const Started: Story = {
  args: {
    title: "Title",
    status: "STARTED",
    startTime: times[0],
    aTeamName: "ATeam",
    bTeamName: "BTeam",
    aTeamAvatarSrc: images[0],
    bTeamAvatarSrc: images[0],
    aTeamScore: 10,
    bTeamScore: 30,
    anchors: [
      {
        name: "Allie Ferguson",
        src: images[1],
      },
      {
        src: images[1],
      },
      {
        name: "Roy Schultz",
      },

      {
        name: "Jonathan Thomas",
        src: images[1],
      },
      {
        name: "Luke Osborne",
        src: images[1],
      },
      {
        name: "Belle Williamson",
        src: images[1],
      },
      {
        name: "Effie Castro",
        src: images[1],
      },
      {
        name: "Blake French",
        src: images[1],
      },
      {
        name: "Isaiah Kelly",
        src: images[1],
      },
    ],
  },
};

export const Ending: Story = {
  args: {
    title: "Title",
    status: "ENDING",
    startTime: times[0],
    aTeamName: "ATeam",
    bTeamName: "BTeam",
    aTeamAvatarSrc: images[0],
    bTeamAvatarSrc: images[0],
    aTeamScore: 10,
    bTeamScore: 30,
    anchors: [
      {
        name: "Allie Ferguson",
        src: images[1],
      },
      {
        src: images[1],
      },
      {
        name: "Roy Schultz",
      },

      {
        name: "Jonathan Thomas",
        src: images[1],
      },
      {
        name: "Luke Osborne",
        src: images[1],
      },
      {
        name: "Belle Williamson",
        src: images[1],
      },
      {
        name: "Effie Castro",
        src: images[1],
      },
      {
        name: "Blake French",
        src: images[1],
      },
      {
        name: "Isaiah Kelly",
        src: images[1],
      },
    ],
  },
};
