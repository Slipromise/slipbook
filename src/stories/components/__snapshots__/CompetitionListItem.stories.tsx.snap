// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`stories/components/CompetitionListItem Ending smoke-test 1`] = `
<div data-status="ENDING"
     class="CompetitionListItem_container__3sfl6 hstack"
>
  <div data-header="true"
       class="vstack"
  >
    <span data-title="true">
      Title
    </span>
    <span data-time="true">
      19:33
    </span>
  </div>
  <div data-body="true"
       class="vstack"
  >
    <div data-team="true"
         data-winner="false"
         data-is-tie="false"
         class="vstack"
    >
      <div class="hstack">
        <span data-team-name="true">
          ATeam
        </span>
        <img data-team-logo="true"
             src="data:image/svg+xml;charset=UTF-8,%3Csvg%20width%3D%2264%22%20height%3D%2264%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%2064%2064%22%20preserveAspectRatio%3D%22none%22%3E%3Cdefs%3E%3Cstyle%20type%3D%22text%2Fcss%22%3E%23holder_18f57ed3b02%20text%20%7B%20fill%3A%231C2846%3Bfont-weight%3Abold%3Bfont-family%3AArial%2C%20Helvetica%2C%20Open%20Sans%2C%20sans-serif%2C%20monospace%3Bfont-size%3A10pt%20%7D%20%3C%2Fstyle%3E%3C%2Fdefs%3E%3Cg%20id%3D%22holder_18f57ed3b02%22%3E%3Crect%20width%3D%2264%22%20height%3D%2264%22%20fill%3D%22%23F8591A%22%3E%3C%2Frect%3E%3Cg%3E%3Ctext%20x%3D%2213.4609375%22%20y%3D%2236.5%22%3E64x64%3C%2Ftext%3E%3C%2Fg%3E%3C%2Fg%3E%3C%2Fsvg%3E"
             alt="Home Team"
             class
        >
      </div>
      <span data-score="true">
        10
      </span>
    </div>
    <svg stroke="currentColor"
         fill="none"
         stroke-width="2"
         viewbox="0 0 24 24"
         stroke-linecap="round"
         stroke-linejoin="round"
         class="CompetitionListItem_vs-icon__YBcw3"
         height="2em"
         width="2em"
         xmlns="http://www.w3.org/2000/svg"
    >
      <polyline points="14.5 17.5 3 6 3 3 6 3 17.5 14.5">
      </polyline>
      <line x1="13"
            x2="19"
            y1="19"
            y2="13"
      >
      </line>
      <line x1="16"
            x2="20"
            y1="16"
            y2="20"
      >
      </line>
      <line x1="19"
            x2="21"
            y1="21"
            y2="19"
      >
      </line>
      <polyline points="14.5 6.5 18 3 21 3 21 6 17.5 9.5">
      </polyline>
      <line x1="5"
            x2="9"
            y1="14"
            y2="18"
      >
      </line>
      <line x1="7"
            x2="4"
            y1="17"
            y2="20"
      >
      </line>
      <line x1="3"
            x2="5"
            y1="19"
            y2="21"
      >
      </line>
    </svg>
    <div data-team="true"
         data-winner="true"
         data-is-tie="false"
         class="vstack"
    >
      <div class="hstack">
        <img data-team-logo="true"
             src="data:image/svg+xml;charset=UTF-8,%3Csvg%20width%3D%2264%22%20height%3D%2264%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%2064%2064%22%20preserveAspectRatio%3D%22none%22%3E%3Cdefs%3E%3Cstyle%20type%3D%22text%2Fcss%22%3E%23holder_18f57ed3b02%20text%20%7B%20fill%3A%231C2846%3Bfont-weight%3Abold%3Bfont-family%3AArial%2C%20Helvetica%2C%20Open%20Sans%2C%20sans-serif%2C%20monospace%3Bfont-size%3A10pt%20%7D%20%3C%2Fstyle%3E%3C%2Fdefs%3E%3Cg%20id%3D%22holder_18f57ed3b02%22%3E%3Crect%20width%3D%2264%22%20height%3D%2264%22%20fill%3D%22%23F8591A%22%3E%3C%2Frect%3E%3Cg%3E%3Ctext%20x%3D%2213.4609375%22%20y%3D%2236.5%22%3E64x64%3C%2Ftext%3E%3C%2Fg%3E%3C%2Fg%3E%3C%2Fsvg%3E"
             alt="Away Team"
             class
        >
        <span data-team-name="true">
          BTeam
        </span>
      </div>
      <span data-score="true">
        30
      </span>
    </div>
  </div>
  <div data-footer="true"
       data-is-collapsed="true"
       class="vstack"
  >
    <figure class="AnchorAvatar_container__AvuZi figure">
      <img src="data:image/svg+xml;charset=UTF-8,%3Csvg%20width%3D%2280%22%20height%3D%2280%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%2080%2080%22%20preserveAspectRatio%3D%22none%22%3E%3Cdefs%3E%3Cstyle%20type%3D%22text%2Fcss%22%3E%23holder_18f57f77702%20text%20%7B%20fill%3A%23FFFFFF%3Bfont-weight%3Abold%3Bfont-family%3AArial%2C%20Helvetica%2C%20Open%20Sans%2C%20sans-serif%2C%20monospace%3Bfont-size%3A10pt%20%7D%20%3C%2Fstyle%3E%3C%2Fdefs%3E%3Cg%20id%3D%22holder_18f57f77702%22%3E%3Crect%20width%3D%2280%22%20height%3D%2280%22%20fill%3D%22%233a5a97%22%3E%3C%2Frect%3E%3Cg%3E%3Ctext%20x%3D%2221.4609375%22%20y%3D%2244.5%22%3E80x80%3C%2Ftext%3E%3C%2Fg%3E%3C%2Fg%3E%3C%2Fsvg%3E"
           alt
           class="figure-img img-fluid"
      >
      <figcaption class="figure-caption">
        Allie Ferguson
      </figcaption>
    </figure>
    <figure class="AnchorAvatar_container__AvuZi figure">
      <img src="data:image/svg+xml;charset=UTF-8,%3Csvg%20width%3D%2280%22%20height%3D%2280%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%2080%2080%22%20preserveAspectRatio%3D%22none%22%3E%3Cdefs%3E%3Cstyle%20type%3D%22text%2Fcss%22%3E%23holder_18f57f77702%20text%20%7B%20fill%3A%23FFFFFF%3Bfont-weight%3Abold%3Bfont-family%3AArial%2C%20Helvetica%2C%20Open%20Sans%2C%20sans-serif%2C%20monospace%3Bfont-size%3A10pt%20%7D%20%3C%2Fstyle%3E%3C%2Fdefs%3E%3Cg%20id%3D%22holder_18f57f77702%22%3E%3Crect%20width%3D%2280%22%20height%3D%2280%22%20fill%3D%22%233a5a97%22%3E%3C%2Frect%3E%3Cg%3E%3Ctext%20x%3D%2221.4609375%22%20y%3D%2244.5%22%3E80x80%3C%2Ftext%3E%3C%2Fg%3E%3C%2Fg%3E%3C%2Fsvg%3E"
           alt
           class="figure-img img-fluid"
      >
      <figcaption class="figure-caption">
      </figcaption>
    </figure>
    <figure class="AnchorAvatar_container__AvuZi figure">
      <div data-default-image="true">
        <span>
          R
        </span>
      </div>
      <figcaption class="figure-caption">
        Roy Schultz
      </figcaption>
    </figure>
    <figure class="AnchorAvatar_container__AvuZi figure">
      <img src="data:image/svg+xml;charset=UTF-8,%3Csvg%20width%3D%2280%22%20height%3D%2280%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%2080%2080%22%20preserveAspectRatio%3D%22none%22%3E%3Cdefs%3E%3Cstyle%20type%3D%22text%2Fcss%22%3E%23holder_18f57f77702%20text%20%7B%20fill%3A%23FFFFFF%3Bfont-weight%3Abold%3Bfont-family%3AArial%2C%20Helvetica%2C%20Open%20Sans%2C%20sans-serif%2C%20monospace%3Bfont-size%3A10pt%20%7D%20%3C%2Fstyle%3E%3C%2Fdefs%3E%3Cg%20id%3D%22holder_18f57f77702%22%3E%3Crect%20width%3D%2280%22%20height%3D%2280%22%20fill%3D%22%233a5a97%22%3E%3C%2Frect%3E%3Cg%3E%3Ctext%20x%3D%2221.4609375%22%20y%3D%2244.5%22%3E80x80%3C%2Ftext%3E%3C%2Fg%3E%3C%2Fg%3E%3C%2Fsvg%3E"
           alt
           class="figure-img img-fluid"
      >
      <figcaption class="figure-caption">
        Jonathan Thomas
      </figcaption>
    </figure>
    <div class="CompetitionListItem_more__Hqkq1">
      <span>
        +5
      </span>
    </div>
  </div>
</div>
`;

exports[`stories/components/CompetitionListItem Normal smoke-test 1`] = `
<div data-status="NONE"
     class="CompetitionListItem_container__3sfl6 hstack"
>
  <div data-header="true"
       class="vstack"
  >
    <span data-title="true">
      Title
    </span>
    <span data-time="true">
      19:33
    </span>
  </div>
  <div data-body="true"
       class="vstack"
  >
    <div data-team="true"
         data-winner="false"
         data-is-tie="false"
         class="vstack"
    >
      <div class="hstack">
        <span data-team-name="true">
          ATeam
        </span>
        <img data-team-logo="true"
             src="data:image/svg+xml;charset=UTF-8,%3Csvg%20width%3D%2264%22%20height%3D%2264%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%2064%2064%22%20preserveAspectRatio%3D%22none%22%3E%3Cdefs%3E%3Cstyle%20type%3D%22text%2Fcss%22%3E%23holder_18f57ed3b02%20text%20%7B%20fill%3A%231C2846%3Bfont-weight%3Abold%3Bfont-family%3AArial%2C%20Helvetica%2C%20Open%20Sans%2C%20sans-serif%2C%20monospace%3Bfont-size%3A10pt%20%7D%20%3C%2Fstyle%3E%3C%2Fdefs%3E%3Cg%20id%3D%22holder_18f57ed3b02%22%3E%3Crect%20width%3D%2264%22%20height%3D%2264%22%20fill%3D%22%23F8591A%22%3E%3C%2Frect%3E%3Cg%3E%3Ctext%20x%3D%2213.4609375%22%20y%3D%2236.5%22%3E64x64%3C%2Ftext%3E%3C%2Fg%3E%3C%2Fg%3E%3C%2Fsvg%3E"
             alt="Home Team"
             class
        >
      </div>
      <span data-score="true">
      </span>
    </div>
    <svg stroke="currentColor"
         fill="none"
         stroke-width="2"
         viewbox="0 0 24 24"
         stroke-linecap="round"
         stroke-linejoin="round"
         class="CompetitionListItem_vs-icon__YBcw3"
         height="2em"
         width="2em"
         xmlns="http://www.w3.org/2000/svg"
    >
      <polyline points="14.5 17.5 3 6 3 3 6 3 17.5 14.5">
      </polyline>
      <line x1="13"
            x2="19"
            y1="19"
            y2="13"
      >
      </line>
      <line x1="16"
            x2="20"
            y1="16"
            y2="20"
      >
      </line>
      <line x1="19"
            x2="21"
            y1="21"
            y2="19"
      >
      </line>
      <polyline points="14.5 6.5 18 3 21 3 21 6 17.5 9.5">
      </polyline>
      <line x1="5"
            x2="9"
            y1="14"
            y2="18"
      >
      </line>
      <line x1="7"
            x2="4"
            y1="17"
            y2="20"
      >
      </line>
      <line x1="3"
            x2="5"
            y1="19"
            y2="21"
      >
      </line>
    </svg>
    <div data-team="true"
         data-winner="true"
         data-is-tie="false"
         class="vstack"
    >
      <div class="hstack">
        <img data-team-logo="true"
             src="data:image/svg+xml;charset=UTF-8,%3Csvg%20width%3D%2264%22%20height%3D%2264%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%2064%2064%22%20preserveAspectRatio%3D%22none%22%3E%3Cdefs%3E%3Cstyle%20type%3D%22text%2Fcss%22%3E%23holder_18f57ed3b02%20text%20%7B%20fill%3A%231C2846%3Bfont-weight%3Abold%3Bfont-family%3AArial%2C%20Helvetica%2C%20Open%20Sans%2C%20sans-serif%2C%20monospace%3Bfont-size%3A10pt%20%7D%20%3C%2Fstyle%3E%3C%2Fdefs%3E%3Cg%20id%3D%22holder_18f57ed3b02%22%3E%3Crect%20width%3D%2264%22%20height%3D%2264%22%20fill%3D%22%23F8591A%22%3E%3C%2Frect%3E%3Cg%3E%3Ctext%20x%3D%2213.4609375%22%20y%3D%2236.5%22%3E64x64%3C%2Ftext%3E%3C%2Fg%3E%3C%2Fg%3E%3C%2Fsvg%3E"
             alt="Away Team"
             class
        >
        <span data-team-name="true">
          BTeam
        </span>
      </div>
      <span data-score="true">
      </span>
    </div>
  </div>
</div>
`;

exports[`stories/components/CompetitionListItem Started smoke-test 1`] = `
<div data-status="STARTED"
     class="CompetitionListItem_container__3sfl6 hstack"
>
  <div data-header="true"
       class="vstack"
  >
    <span data-title="true">
      Title
    </span>
    <span data-time="true">
      19:33
    </span>
  </div>
  <div data-body="true"
       class="vstack"
  >
    <div data-team="true"
         data-winner="false"
         data-is-tie="false"
         class="vstack"
    >
      <div class="hstack">
        <span data-team-name="true">
          ATeam
        </span>
        <img data-team-logo="true"
             src="data:image/svg+xml;charset=UTF-8,%3Csvg%20width%3D%2264%22%20height%3D%2264%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%2064%2064%22%20preserveAspectRatio%3D%22none%22%3E%3Cdefs%3E%3Cstyle%20type%3D%22text%2Fcss%22%3E%23holder_18f57ed3b02%20text%20%7B%20fill%3A%231C2846%3Bfont-weight%3Abold%3Bfont-family%3AArial%2C%20Helvetica%2C%20Open%20Sans%2C%20sans-serif%2C%20monospace%3Bfont-size%3A10pt%20%7D%20%3C%2Fstyle%3E%3C%2Fdefs%3E%3Cg%20id%3D%22holder_18f57ed3b02%22%3E%3Crect%20width%3D%2264%22%20height%3D%2264%22%20fill%3D%22%23F8591A%22%3E%3C%2Frect%3E%3Cg%3E%3Ctext%20x%3D%2213.4609375%22%20y%3D%2236.5%22%3E64x64%3C%2Ftext%3E%3C%2Fg%3E%3C%2Fg%3E%3C%2Fsvg%3E"
             alt="Home Team"
             class
        >
      </div>
      <span data-score="true">
        10
      </span>
    </div>
    <svg stroke="currentColor"
         fill="none"
         stroke-width="2"
         viewbox="0 0 24 24"
         stroke-linecap="round"
         stroke-linejoin="round"
         class="CompetitionListItem_vs-icon__YBcw3"
         height="2em"
         width="2em"
         xmlns="http://www.w3.org/2000/svg"
    >
      <polyline points="14.5 17.5 3 6 3 3 6 3 17.5 14.5">
      </polyline>
      <line x1="13"
            x2="19"
            y1="19"
            y2="13"
      >
      </line>
      <line x1="16"
            x2="20"
            y1="16"
            y2="20"
      >
      </line>
      <line x1="19"
            x2="21"
            y1="21"
            y2="19"
      >
      </line>
      <polyline points="14.5 6.5 18 3 21 3 21 6 17.5 9.5">
      </polyline>
      <line x1="5"
            x2="9"
            y1="14"
            y2="18"
      >
      </line>
      <line x1="7"
            x2="4"
            y1="17"
            y2="20"
      >
      </line>
      <line x1="3"
            x2="5"
            y1="19"
            y2="21"
      >
      </line>
    </svg>
    <div data-team="true"
         data-winner="true"
         data-is-tie="false"
         class="vstack"
    >
      <div class="hstack">
        <img data-team-logo="true"
             src="data:image/svg+xml;charset=UTF-8,%3Csvg%20width%3D%2264%22%20height%3D%2264%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%2064%2064%22%20preserveAspectRatio%3D%22none%22%3E%3Cdefs%3E%3Cstyle%20type%3D%22text%2Fcss%22%3E%23holder_18f57ed3b02%20text%20%7B%20fill%3A%231C2846%3Bfont-weight%3Abold%3Bfont-family%3AArial%2C%20Helvetica%2C%20Open%20Sans%2C%20sans-serif%2C%20monospace%3Bfont-size%3A10pt%20%7D%20%3C%2Fstyle%3E%3C%2Fdefs%3E%3Cg%20id%3D%22holder_18f57ed3b02%22%3E%3Crect%20width%3D%2264%22%20height%3D%2264%22%20fill%3D%22%23F8591A%22%3E%3C%2Frect%3E%3Cg%3E%3Ctext%20x%3D%2213.4609375%22%20y%3D%2236.5%22%3E64x64%3C%2Ftext%3E%3C%2Fg%3E%3C%2Fg%3E%3C%2Fsvg%3E"
             alt="Away Team"
             class
        >
        <span data-team-name="true">
          BTeam
        </span>
      </div>
      <span data-score="true">
        30
      </span>
    </div>
  </div>
  <div data-footer="true"
       data-is-collapsed="true"
       class="vstack"
  >
    <figure class="AnchorAvatar_container__AvuZi figure">
      <img src="data:image/svg+xml;charset=UTF-8,%3Csvg%20width%3D%2280%22%20height%3D%2280%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%2080%2080%22%20preserveAspectRatio%3D%22none%22%3E%3Cdefs%3E%3Cstyle%20type%3D%22text%2Fcss%22%3E%23holder_18f57f77702%20text%20%7B%20fill%3A%23FFFFFF%3Bfont-weight%3Abold%3Bfont-family%3AArial%2C%20Helvetica%2C%20Open%20Sans%2C%20sans-serif%2C%20monospace%3Bfont-size%3A10pt%20%7D%20%3C%2Fstyle%3E%3C%2Fdefs%3E%3Cg%20id%3D%22holder_18f57f77702%22%3E%3Crect%20width%3D%2280%22%20height%3D%2280%22%20fill%3D%22%233a5a97%22%3E%3C%2Frect%3E%3Cg%3E%3Ctext%20x%3D%2221.4609375%22%20y%3D%2244.5%22%3E80x80%3C%2Ftext%3E%3C%2Fg%3E%3C%2Fg%3E%3C%2Fsvg%3E"
           alt
           class="figure-img img-fluid"
      >
      <figcaption class="figure-caption">
        Allie Ferguson
      </figcaption>
    </figure>
    <figure class="AnchorAvatar_container__AvuZi figure">
      <img src="data:image/svg+xml;charset=UTF-8,%3Csvg%20width%3D%2280%22%20height%3D%2280%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%2080%2080%22%20preserveAspectRatio%3D%22none%22%3E%3Cdefs%3E%3Cstyle%20type%3D%22text%2Fcss%22%3E%23holder_18f57f77702%20text%20%7B%20fill%3A%23FFFFFF%3Bfont-weight%3Abold%3Bfont-family%3AArial%2C%20Helvetica%2C%20Open%20Sans%2C%20sans-serif%2C%20monospace%3Bfont-size%3A10pt%20%7D%20%3C%2Fstyle%3E%3C%2Fdefs%3E%3Cg%20id%3D%22holder_18f57f77702%22%3E%3Crect%20width%3D%2280%22%20height%3D%2280%22%20fill%3D%22%233a5a97%22%3E%3C%2Frect%3E%3Cg%3E%3Ctext%20x%3D%2221.4609375%22%20y%3D%2244.5%22%3E80x80%3C%2Ftext%3E%3C%2Fg%3E%3C%2Fg%3E%3C%2Fsvg%3E"
           alt
           class="figure-img img-fluid"
      >
      <figcaption class="figure-caption">
      </figcaption>
    </figure>
    <figure class="AnchorAvatar_container__AvuZi figure">
      <div data-default-image="true">
        <span>
          R
        </span>
      </div>
      <figcaption class="figure-caption">
        Roy Schultz
      </figcaption>
    </figure>
    <figure class="AnchorAvatar_container__AvuZi figure">
      <img src="data:image/svg+xml;charset=UTF-8,%3Csvg%20width%3D%2280%22%20height%3D%2280%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%2080%2080%22%20preserveAspectRatio%3D%22none%22%3E%3Cdefs%3E%3Cstyle%20type%3D%22text%2Fcss%22%3E%23holder_18f57f77702%20text%20%7B%20fill%3A%23FFFFFF%3Bfont-weight%3Abold%3Bfont-family%3AArial%2C%20Helvetica%2C%20Open%20Sans%2C%20sans-serif%2C%20monospace%3Bfont-size%3A10pt%20%7D%20%3C%2Fstyle%3E%3C%2Fdefs%3E%3Cg%20id%3D%22holder_18f57f77702%22%3E%3Crect%20width%3D%2280%22%20height%3D%2280%22%20fill%3D%22%233a5a97%22%3E%3C%2Frect%3E%3Cg%3E%3Ctext%20x%3D%2221.4609375%22%20y%3D%2244.5%22%3E80x80%3C%2Ftext%3E%3C%2Fg%3E%3C%2Fg%3E%3C%2Fsvg%3E"
           alt
           class="figure-img img-fluid"
      >
      <figcaption class="figure-caption">
        Jonathan Thomas
      </figcaption>
    </figure>
    <div class="CompetitionListItem_more__Hqkq1">
      <span>
        +5
      </span>
    </div>
  </div>
</div>
`;
